vhdbuilder/prefetch/internal/containerimage/containerimage_test.go

```diff
@@ -13,23 +14,25 @@ import (
 
 const (
 	testDataPath       = "testdata/"
+	artifactsRelPath   = "parts/linux/cloud-init/artifacts"
 	regenerateTestData = "REGENERATE_CONTAINER_IMAGE_PREFETCH_TESTDATA"
 )
 
 var (
-	componentsTestDataPath     = filepath.Join(testDataPath, "components.json")
 	prefetchScriptTestDataPath = filepath.Join(testDataPath, "prefetch.sh")
 )
 
 func TestContianerImage(t *testing.T) {
```

Comment:
Typo

Reply:
`TestContainerImage` aligns with the name of the package name, `containerimage`